test/Microsoft.TemplateEngine.Edge.UnitTests/PathInfoTests.cs

```diff
@@ -14,6 +14,63 @@ namespace Microsoft.TemplateEngine.Edge.UnitTests
 {
     public class PathInfoTests
     {
+        [Theory]
+        [InlineData (false, false, false)]
+        [InlineData (false, false, true)]
+        [InlineData (false, true, false)]
+        [InlineData (false, true, true)]
+        [InlineData (true, false, false)]
+        [InlineData (true, false, true)]
+        [InlineData (true, true, false)]
+        [InlineData (true, true, true)]
+        public void UserProfileEnvironment(bool useHome, bool useDotnetCliHome, bool useUserProfilePath)
```

Comment:
this test might not be relevant anymore. Instead please consider unit test in `Cli.UnitTests` or integration test in `dotnet-new3.IntegrationTests`. 

Here is an example of integration test, which modifies environment variables before act: https://github.com/dotnet/templating/blob/209d566999ffc0f89deb5948b022c4331715fb1b/test/dotnet-new3.UnitTests/DotnetNewDebugOptions.cs#L103-L115